bamboo/set-bamboo-env-variables.sh

```diff
@@ -142,6 +119,35 @@ if [[ $DEPLOYMENT =~ '-tf' ]]; then
   export AWS_SECRET_ACCESS_KEY=$bamboo_SECRET_NONPROD_AWS_SECRET_ACCESS_KEY
 fi
 
+export PR_BRANCH=master
+if [[  $DEPLOYMENT =~ '-tf' ]]; then
+  echo "Setting GIT_PR target branch to 'terraform'"
+  export $PR_BRANCH=terraform
+fi
+
+## Run detect-pr script and set flag to true/false
```

Comment:
The block below L128  hasn't changed, just reordering so L122-128 can run in the right sequence